dependencies.props

```diff
@@ -7,8 +7,14 @@
 
        These ref versions are pulled from https://github.com/dotnet/versions.
   -->
+
   <PropertyGroup>
-    <BuildToolsCurrentRef>044092b54a7e4df96d059e2c6b83463714547bf8</BuildToolsCurrentRef>
+    <!-- Opt-out repo features -->
+    <UsingToolXliff>false</UsingToolXliff>
```

Comment:
Why does this need to be in both dependencies.props and version.props?  Eventually we should combine these two files into version.props

Reply:
It doesn't, looks like an oversight. Will fix.